alicloud/service_alicloud_rkv.go

```diff
@@ -0,0 +1,58 @@
+package alicloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/r-kvstore"
+	"github.com/denverdino/aliyungo/common"
+)
+
+func (client *AliyunClient) DescribeRKVInstanceById(id string) (instance *r_kvstore.DBInstanceAttribute, err error) {
+	request := r_kvstore.CreateDescribeInstanceAttributeRequest()
+	request.InstanceId = id
+	resp, err := client.rkvconn.DescribeInstanceAttribute(request)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I think, if the error code is InvalidRKVInstanceIdNotFound and it can be used to build a NotFoundError.

Reply:
See below comment.